### DISCUSSION TITLE
PiJuice Zero Telemetry Data Remains Constant

### DISCUSSION THREAD

**Discussion by baqwas** [PiSupply/PiJuice#1053] [Q&A] [answered]
A _newbie_ here! Will gladly RTM if you can tell me which one!!

Quick question, please:

I've been using the Python interface to log some values from a **PiJuice Zero** (on a newly acquired 02W after waiting for nearly 2 years!). The readings do not change using the _PiSupply 1200 mAH battery_ based on 48 hours worth of observations:
- _batteryCharge_ 87
- _batteryVoltage_ 4171
- _batteryTemperature_ 25
- _ioVoltage_ 5070
- _ioCurrent_ 580 (sometimes 570)

There is no active workload on the RPi02W (Bullseye with all _updates_ applied) but I would expect that the _batteryCharge_ should saturate to a higher value? Also, the temperature is relatively constant. Others temperature sensors in the room do change ever so lightly throughout the day but the PiJuice  numbers stays constant.

The Python script is the simplest call to the corresponding functions as derived from code snippets in the test examples in this repository.

Of course, I may be extrapolating to an erroneous conclusion here by drawing on my UPS telemetry experience but I hope that someone can help me understand which parameters should be monitored (apart from the status). Thanks.

Kind regards.

**Comment by tvoverbeek** [comment, marked as answer]
Can you run pijuice_cli? If so, please provide a screenshot of the status page:
<img width="513" alt="Screenshot 2024-01-03 at 21 11 22" src="https://github.com/PiSupply/PiJuice/assets/1197294/6699458f-045d-4e63-b76f-6f23ef44c309">

**Reply by baqwas** [reply]
![Pasted image](https://github.com/PiSupply/PiJuice/assets/7071255/18fa3c6e-e7cd-48e7-8039-8af2ad590173)

Thanks, @tvoverbeek! The ioCurrent changes by two decimal places and the ioVoltage only occasionally by the last digit. Makes sense since there is hardly any load. Everything else remains constant. Also, perhaps I'm misunderstanding the batteryTemperature value. Perhaps I was too hasty.

Regards.
